Add mirror sensors to KBOMirrorHEStates

nrwslac opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->
## Description
<!--- Describe your changes in detail -->
- add mirror sensor component to `KBOMirrorHE`
- this will affect:
```
| name             | mr3k2_kbh
| name             | mr4k2_kbv
| name             | mr2k4_kbo
| name             | mr3k4_kbo

```
All tested locally.
## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here. -->
- RTDs installed on mirror

## How Has This Been Tested?
<!--- Please describe in detail how you tested your changes. -->
<!--- Include details of your testing environment, and the tests you ran to -->
<!--- see how your change affects other areas of the code, etc. -->
- ran locally and launched screens
## Where Has This Been Documented?
<!--  Include where the changes made have been documented. -->
<!--  This can simply be  a comment in the code or updating a docstring -->
https://jira.slac.stanford.edu/browse/ECS-6616
https://jira.slac.stanford.edu/browse/ECS-6705
<!--
## Screenshots (if appropriate):
-->
![cooling-readouts](https://github.com/user-attachments/assets/b0b62cca-2eac-4e7a-a739-4472d836e121)
## Pre-merge checklist
- [ ] Code works interactively
- [ ] Code contains descriptive docstrings, including context and API
- [ ] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on GitHub Actions
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

tangkong commented on docs/source/upcoming_release_notes/1304-chin-rtds.rst:
```diff
@@ -0,0 +1,30 @@
+1304 chin-rtds
```
like the chin on my face?

nrwslac replied:
wait till you meet the mirrors with a tail.

tangkong reviewed (commented):
I have this mental image of mirrors being little faces with chin-guards now.  

A few nitpicks here, and I do want to call out that the component names have changed, breaking the previous interface.  I expect this to be fine, but have no real way of knowing or testing this (read: I don't know what's out in the field that may rely on these devices)

nrwslac commented on docs/source/upcoming_release_notes/1304-chin-rtds.rst:
```diff
@@ -0,0 +1,30 @@
+
+API Breaks
+----------
+- N/A
```
This is true. In this case, we should be clear. The `FFMirrorZ` mirrors are limited to MR4/5K4KBO in TMO; they haven't been fully commissioned yet afaik. I'll update the docs.

nrwslac replied:
update this as an API break in detail.

nrwslac commented on docs/source/upcoming_release_notes/1304-chin-rtds.rst:
```diff
@@ -0,0 +1,30 @@
+
+Bugfixes
+--------
+- reorder some components in `KBOMirrorHE`
```
Moved to maintenance and added more details.

nrwslac commented:
> I have this mental image of mirrors being little faces with chin-guards now.
> 
> A few nitpicks here, and I do want to call out that the component names have changed, breaking the previous interface. I expect this to be fine, but have no real way of knowing or testing this (read: I don't know what's out in the field that may rely on these devices)

I think I addressed all concerns here, thank you for reviewing!

ZLLentz commented:
I greatly appreciate your inclusion of the screenshots into the pr description

ZLLentz commented on pcdsdevices/mirror.py:
```diff
@@ -1154,6 +1154,10 @@ class KBOMirrorHE(KBOMirror):
     cool_flow1 = Cpt(EpicsSignalRO, ':FWM:1_RBV', kind='normal')
     cool_press = Cpt(EpicsSignalRO, ':PRSM:1_RBV', kind='normal')
 
+    #
```
nitpick
```suggestion
```

nrwslac replied:
Where'd this little dude come from. Updated

tangkong reviewed (approved):
👍
